### PR TITLE
Fix adapters

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/algofi/index.js exports TVL: 
        
 
 
    algorand-borrowed         58.99 M
    algorand                  93.61 M
    algorand-staking          12.71 M
    borrowed                  58.99 M
    staking                   12.71 M
    
    total                    93.61 M

**Comment by llamatester** [comment]
The adapter at projects/firebird/index.js exports TVL: 
        
 
 
    polygon                   1.92 M
    
    total                    1.92 M

**Comment by llamatester** [comment]
The adapter at projects/yogi/index.js exports TVL: 
        
 
 
    polygon                   4.17 M
    bsc                       12.93 k
    
    total                    4.19 M

File: projects/algofi/index.js [hunk 1]
```diff
@@ -261,7 +261,7 @@ async function dex() {
           async (bail) =>
             await axios.get("https://thf1cmidt1.execute-api.us-east-2.amazonaws.com/Prod/amm_protocol_snapshot/?network=MAINNET")
         )
-      ).data.asset_snapshots[0].tvl;
+      ).data.protocol_snapshot.tvl.total_usd
```
Comment (0xngmi) [inline projects/algofi/index.js:264]: reee

**Pull request merged**